README.md

```diff
@@ -2,11 +2,38 @@
 
 This repository is the source to the Bottlerocket project website. It contains the content (as [Markdown](https://commonmark.org/) files), templates (HTML), stylesheets (CSS), and [Hugo](https://gohugo.io/) configurations (mostly [TOML](https://toml.io/en/), some [YAML](https://yaml.org/)).
 
-
 ## Contributing
 
 The Bottlerocket website is open to contribution and collaboration. See [CONTRIBUTING](CONTRIBUTING.md) for specific instructions.
 
+### Rendering Locally
+
+This site uses [Hugo](https://github.com/gohugoio/hugo) for rendering.
+You may run `hugo` locally to validate your changes render properly.
+
+#### Rendering
+
+If [`hugo` is installed](#hugo-installation), you may view the rendered site by running:
+
+```sh
+hugo server -D
```

Comment:
Why render drafts (`-D`)?

Reply:
If you are making local changes then you may want to view those changes while still in draft form. This would be a difference between the public publishing of the site versus locally viewing you changes.